### DISCUSSION TITLE
Quick edit - custom UI 10

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14683] [How to] [answered]
May I ask is there a way to show little pencil on mouse over in custom reference fields?

**Comment by rskrzypczak** [comment, marked as answer]
Quick Reference Field Editing is not available.

**Comment by mariuszkrzaczkowski** [comment]
look at this
![image](https://user-images.githubusercontent.com/10198654/118606607-ed1e0d80-b7b7-11eb-96b9-7926bd751444.png)
